ui/lib/core/app/components/radio-card.js

```diff
@@ -0,0 +1 @@
+export { default } from 'core/components/radio-card';
```

Comment:
We'll need this component in pki engine as well. We're still a ways off from working on the config, but could this change be merged into main?

Reply:
Yes I will work on that after this PR is merged.